code/src/java/pcgen/rules/persistence/util/TokenFamily.java

```diff
@@ -280,4 +283,24 @@ public void addDeferredToken(DeferredToken<?> newToken)
 	{
 		deferredTokenList.add(newToken);
 	}
+
+	public void addGroupDefinition(GroupDefinition<?> def)

```

Comment:
I didn't understand the javadoc for GroupDefinition, so it would be good to explain its use here in a more concrete manner.